website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,894 @@
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community. Recently, Metals has also gained
+support from [Lunatech](https://lunatech.com) with the contributions from
+@ckipp01. We are extremely grateful for the growing support from the community
+and businesses alike, which shows in the number of new contributions and their
+quality.
+
+In this release we merged 207 PRs and closed 82 issues, full details:
+https://github.com/scalameta/metals/milestone/9?closed=1
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- go to implementations
+- rename symbol
+- first code action - import missing symbol
+- support for Pants build tool
+- run and debug for tests and main methods
+- improvements for Bloop integration
+- worksheet support
+- better support for Vim via coc-metals
+- completion to add all abstract members
+
+## Navigation
+
+### Go to implementations
+
+Additional index was created inside of Metals that keeps up-to-date information
+about class hierarchies inside the workspace based on the SemanticDB
+information. This allows us to find both all classes extending the particular
+parent class and symbols being implemented inside of those classes. Due to usage
+of semanticDB, this will work after the workspace is fully compiled.
+
+![implementations](https://i.imgur.com/rke6iny.gif)
+
+Finding implementation also works for symbols outside of the workspace, although
+in that case it might take a while to load the classpath symbol information at
+first. Consecutive calls will reuse cached classpath symbols information, so
+this might take longer only once per build target.
+
+## Refactor
+
+### Rename symbol
+
+Thanks to implementing the `Go to implementations` feature, we were able to work
+on a proper rename for any workspace symbol that will rename all occurences of a
+symbol even if that symbol is a part of a class hierarchy.
+
+![rename](https://i.imgur.com/QHaJCr0.gif)
+
+There are a couple of cases in which rename might behave differently based on
+the particular symbol being renamed:
+
+- renaming an overriden symbol will rename all occurrences for both parent and
+  all children overriding that symbol,
+- java symbols are not renamed,
+- renaming "hashCode", "unapply", "unary\_!", "!" is not supported, since they
+  change the semantics,
+- methods starting with `:` can only be renamed to ones with the same first
+  character to make sure code compiles afterwards,
+- renaming apply will change all occurrences with just `()` to a full name
+  `.newname()`,
+- if renaming a class with the same name as the file, the file's name will also
+  be renamed,
+- companion objects will also be renamed with their companion classes.
+
+### Completion to add all abstract members
```

Comment:
Nit: I don't think this should go under "Refactor", since it's a completion item.